mkdocs.yml

```diff
@@ -60,4 +61,5 @@ nav:
           - Stats Calculator: tools/stats_calculator.md
           - HP Calculator: tools/hp_calculator.md
           - MP Calculator: tools/mp_calculator.md
+          - EXP Calculator: tools/exp_calculator.md
```

Comment:
My preference would be to have the `exp_calculator` follow immediately after `stats_calculator` because they're similar.

Reply:
Although probably renamed (see other comments).